clients/drcachesim/tracer/output.cpp

```diff
@@ -728,9 +728,9 @@ output_buffer(void *drcontext, per_thread_t *data, byte *buf_base, byte *buf_ptr
               size_t header_size)
```

Comment:
@derekbruening I'm curious about MAX_NUM_DELAY_ENTRIES and MAX_NUM_DELAY_INSTRS.

```
#define MAX_NUM_DELAY_INSTRS 32
// Really sizeof(trace_entry_t.length)
#define MAX_NUM_DELAY_ENTRIES (MAX_NUM_DELAY_INSTRS / sizeof(addr_t))
```

But doesn't each entry correspond to one instruction in the online mode?

Reply:
There are instruction bundles which combine multiple instructions.

Reply:
Right. But why do we divide by the sizeof?

Reply:
Will follow up offline about what I'm missing.